.github/workflows/ci.yml

```diff
@@ -31,3 +29,27 @@ jobs:
         run: |
           brew install rwx-research/fake-tap-name/${{ matrix.name }}
           abq --version | grep ${{ matrix.version }}
+
+  test_captain:
+    runs-on: macos-11
+    strategy:
+      matrix:
+        include:
+          - name: captain
+            version: "1.5.4"
+          - name: captain@1
+            version: "1.5.4"
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          path: homebrew-fake-tap-name # give it a fake name so we don't accidentally pull the one from github
+
+      - name: move it to the right directory
+        run: |
+          mkdir -p homebrew-tap /usr/local/Homebrew/Library/Taps/rwx-research
+          mv homebrew-fake-tap-name /usr/local/Homebrew/Library/Taps/rwx-research/
+
+      - name: Install captain & run it
+        run: |
+          brew install rwx-research/fake-tap-name/${{ matrix.name }}
+          captain --version | grep ${{ matrix.version }}
```

Comment:
So we'll need to update this workflow every time we publish a new version of Captain to the tap, right?

Reply:
Right